doc/reference/reference_lua/box_index/tuple_pos.rst

```diff
@@ -0,0 +1,54 @@
+.. _box_index-tuple_pos:
+
+===============================================================================
+index_object:tuple_pos()
+===============================================================================
+
+.. class:: index_object
+
+    .. method:: tuple_pos(tuple)
+
+        Return a tuple's position for an index.
+        This value can be passed to the ``after`` option of the ``select`` and ``pairs`` methods:
+
+            *   :ref:`index_object:select <box_index-select>`/:ref:`space_object:select <box_space-select>`
```

Comment:
nit: I'd prefer using `and` instead of the slash. This would improve readability.